sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -2190,6 +2190,12 @@ def expand(self, pcoll):
               'A schema must be provided when writing to BigQuery using '
               'Avro based file loads')
 
+      if self.schema and 'JSON' in str(self.schema):
```

Comment:
hm what if there's a string field called `dataJSON` or something like that? We should probably verify this differently?